src/service/viewer-impl.js

```diff
@@ -742,7 +742,7 @@ export class Viewer {
    * Triggers "documentLoaded" event for the viewer.
    */
   postDocumentReady() {
-    this.sendMessageUnreliable_('documentLoaded', {
+    this.sendMessageUnreliable('documentLoaded', {
```

Comment:
I agree with you that the purpose of `sendMessageUnreliable` is a bit unclear. If you think we can live without the the de-dup logic, then this can be completely gone.

Also want to blame the method for its returned promise, which is indeterministic, caller should not actually rely on it.

I'll change all current usage of `sendMessageUnreliable` just to `sendMessage` if it sounds good to you.